desktop/core/src/desktop/lib/rest/raz_http_client.py

```diff
@@ -116,3 +109,14 @@ def get_sas_token(self, http_method, username, url, params=None, headers=None):
       return response.get('token')
     else:
       raise PopupException(_('No SAS token in RAZ response'), error_code=403)
+
+  def _make_url(self, path, params, do_urlencode=True):
```

Comment:
Can you add a comment here to mentioning why you needed to reimplement this function?

Reply:
Done.